src/wp-admin/includes/class-wp-importer.php

```diff
@@ -289,9 +289,9 @@ public function stop_the_insanity() {
  * Returns value of command line params.
  * Exits when a required param is not set.
  *
- * @param string $param
- * @param bool   $required
- * @return mixed
+ * @param string $param    The parameter name to retrieve.
+ * @param bool   $required Whether the parameter is required. Default false.
+ * @return string|bool|null The parameter value or true if found, null otherwise.
```

Comment:
This function can also `exit` so it can have a "[`never`](https://www.php.net/manual/en/language.types.never.php)" return type:


```suggestion
 * @return string|bool|null|never The parameter value or true if found, null otherwise. The function exits when a required param is missing.
```